int8 type

# enum for geometry type
int8 BOX = 1
int8 SPHERE = 2
int8 CYLINDER = 3
int8 MESH = 4
int8 CAPSULE = 5

float32[] position
float32[] quaternion
float32[] color

string string_data

float32[] float_data